scripts/creation-portal-migration/db/index.js

```
@@ -0,0 +1,52 @@
+const { getDBInstance } = require("./dbConfig");
+const { ObjectId } = require("mongodb");
+const logger = require("../logger");
+
+/**
+* Get all the solution from mongo with the given query
+* @method
+* @name findAll
+* @param {String} clName - collection name
+* @param {Object} query - query to fetch data
+* @returns {JSON}  - returns the collection data
+**/
+const findAll = async (clName, query) => {
+  try {
+    const db = await getDBInstance();
+    return await db
+      .collection(clName)
+      .find({ ...query })
+      .toArray();
+  } catch (err) {
+    logger.error(`findAll Error: , ${err}`);
```

@jayesh9747 why u r not returning error

---

No , need to return the error.